packages/core/src/transport/transport.ts

```diff
@@ -20,9 +20,13 @@ export class HttpRequest {
   send(data: string | FormData, size: number) {
     const url = this.withBatchTime ? addBatchTime(this.endpointUrl) : this.endpointUrl
     if (navigator.sendBeacon && size < this.bytesLimit) {
-      const isQueued = navigator.sendBeacon(url, data)
-      if (isQueued) {
-        return
+      try {
+        const isQueued = navigator.sendBeacon(url, data)
+        if (isQueued) {
+          return
+        }
+      } catch (e) {
+        addErrorToMonitoringBatch(e)
```

Comment:
When `sendBeacon` keeps failing, we might end-up trying to send the error in a loop. This might not be a big issue since it would be throttled by the batch though (and anyway, it wont impact customers since they don't use the internal monitoring).

Reply:
Indeed! I'll update that to send it only once by SDK execution